skrypty/people/bind.lua

```diff
@@ -42,12 +42,12 @@ function scripts.people:enemy_person_build(item)
     if item["name"] ~= "" then
         scripts.people.enemy_suffix[item["name"]] = ""
         if not scripts.people.enemy_suffix[string.lower(item["name"])] then
-            regex = "(^|\\W)((" .. string.sub(item["short"], 0, 1) .. "|" .. first_capitalized .. ")" .. rest_string .. "|" .. item["name"] .. ")(\\W|$)"
+            regex = "(^|\\W)((" .. string.sub(item["short"], 0, 1) .. "|" .. first_capitalized .. ")" .. rest_string .. "|" .. item["name"] .. ")(?! chaosu)(\\W|$)"
```

Comment:
Czy tu nie powinnismy tez dodac `czarnego orka`?

Reply:
To jest inna sytuacja i niestety dosyć ciężka do ugryzienia, chyba, żeby po prostu zrezygnować z subów w przypadku orków, ale wtedy może po prostu z bazy go usunąć i zablokować dodawanie.

Reply:
Chyba tak to bedzie trzeba zrobic, zeby go nie zapisywac wielokrotnie, bo przewaznie recznie go trzeba usuwac.